class TaskError(Exception):
    pass


class InvalidResponse(Exception):
    pass
